### DISCUSSION TITLE
Next Sketchware pro update?

### DISCUSSION THREAD

**Discussion by Fooks-Applications** [Sketchware-Pro/Sketchware-Pro#806] [General]
When is the next Sketchware pro beta gonna release? What kind of cool things will it include

**Comment by PranavPurwar** [comment]
It's not confirmed on when it will be released. It would be released when it would be completed since its the last sketchware pro update (atleast that's what Jbk0 said earlier). Its expected to include stuff like java 11, new local library manager, etc.

**Reply by Fooks-Applications** [reply]
Last update?!?!! Noooo someone has to continue itttt
